WireAPI/Tests/WireAPITests/APIs/ConnectionsAPI/ConnectionsAPITests.swift

```diff
@@ -100,7 +99,7 @@ class ConnectionsAPITests: XCTestCase {
             let response = HTTPClientMock.PredefinedResponse(resourceName: "GetConnectionsMultiplePagesSuccessResponseV0.\(requestIndex)")
             requestIndex += 1
 
-            return HTTPResponse(code: 200, payload: try response.data())
+            return try HTTPResponse(code: 200, payload: response.data())
```

Comment:
Personal preference: disable this rule

Reply:
Do you want me to really disable the rule or change it to format it the opposite direction?

Reply:
If it's possible, I like to have the `try` keyword closest to where it a function throws.

Reply:
Now that I checked again I think it can only be disabled, so I did that.